exchange/adapter_util_test.go

```diff
@@ -279,24 +277,24 @@ func TestBuildExchangeBiddersLegacy(t *testing.T) {
 			description:   "All Active",
 			adapterConfig: map[string]config.Adapter{"lifestreet": cfg, "pulsepoint": cfg},
 			bidderInfos:   map[string]adapters.BidderInfo{"lifestreet": infoActive, "pulsepoint": infoActive},
-			expected:      map[openrtb_ext.BidderName]adaptedBidder{"lifestreet": expectedLifestreet, "pulsepoint": expectedPulsepoint},
+			expected:      map[openrtb_ext.BidderName]adaptedBidder{"lifestreet": expectedLifestreet},
 		},
 		{
 			description:   "All Disabled",
 			adapterConfig: map[string]config.Adapter{"lifestreet": cfg, "pulsepoint": cfg},
-			bidderInfos:   map[string]adapters.BidderInfo{"lifestreet": infoDisabled, "pulsepoint": infoDisabled},
+			bidderInfos:   map[string]adapters.BidderInfo{"lifestreet": infoDisabled},
 			expected:      map[openrtb_ext.BidderName]adaptedBidder{},
 		},
 		{
 			description:   "All Unknown",
 			adapterConfig: map[string]config.Adapter{"lifestreet": cfg, "pulsepoint": cfg},
-			bidderInfos:   map[string]adapters.BidderInfo{"lifestreet": infoUnknown, "pulsepoint": infoUnknown},
+			bidderInfos:   map[string]adapters.BidderInfo{"lifestreet": infoUnknown},
 			expected:      map[openrtb_ext.BidderName]adaptedBidder{},
 		},
 		{
 			description:   "Mixed",
 			adapterConfig: map[string]config.Adapter{"lifestreet": cfg, "pulsepoint": cfg},
-			bidderInfos:   map[string]adapters.BidderInfo{"lifestreet": infoActive, "pulsepoint": infoUnknown},
+			bidderInfos:   map[string]adapters.BidderInfo{"lifestreet": infoActive},
 			expected:      map[openrtb_ext.BidderName]adaptedBidder{"lifestreet": expectedLifestreet},
 		},
 	}
```

Comment:
Could you please also remove:
1. `"pulsepoint": cfg` in the `adapterConfig` setup.
2. The "Mixed" test case.
3. Rename All Active -> Active, All Disabled -> Disabled, All Unknown -> Unkown.

Reply:
fixed